cli/src/main/java/org/infinispan/cli/commands/Config.java

```diff
@@ -156,10 +160,19 @@ public boolean isHelp() {
       public CommandResult exec(ContextAwareCommandInvocation invocation) throws CommandException {
          InputStream is = null;
          OutputStream os = null;
+         ConfigurationResourceResolver resolver;
          try {
             ParserRegistry registry = new ParserRegistry();
+            if (input == null) {
+               is = System.in;
+               resolver = ConfigurationResourceResolver.DEFAULT;
+            } else {
+               File file = new File(input.getAbsolutePath());
+               is = new FileInputStream(file);
+               resolver = new URLConfigurationResourceResolver(file.toURI().toURL());
+            }
             is = input == null ? System.in : new FileInputStream(input.getAbsolutePath());
-            ConfigurationBuilderHolder holder = registry.parse(is, null, null); // Auto-detect type
+            ConfigurationBuilderHolder holder = registry.parse(is, resolver, null); // Auto-detect type
```

Comment:
`is` is being assigned again after the if/else.

Reply:
Fixed